openapi2beans/JavaChecker/src/main/java/dev/galasa/openapi2beans/example/generated/AnEnumProperty.java

```diff
@@ -0,0 +1,7 @@
+package dev.galasa.openapi2beans.example.generated;
+
+// an enum with 2 values to test against.
+public enum AnEnumProperty {
```

Comment:
Hold on. If this is generated, why are we checking it in ? It's in the 'generated' folder...

Reply:
It was originally because it would show red for the tests if the code they're testing doesn't exist, now I'm just unsure :D